usermanagerlib/src/test/java/org/pesmypetcare/usermanagerlib/clients/WeightManagerClientTest.java

```diff
@@ -0,0 +1,143 @@
+package org.pesmypetcare.usermanagerlib.clients;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.pesmypetcare.usermanagerlib.datacontainers.DateTime;
+import org.pesmypetcare.usermanagerlib.datacontainers.Weight;
+import org.pesmypetcare.usermanagerlib.datacontainers.WeightData;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+
+public class WeightManagerClientTest {
+    private static final String ACCESS_TOKEN = "my-token";
+    private static final StringBuilder STATUS_OK = new StringBuilder("200");
+    private static final int STATUS_OK_INT = 200;
+
+    private String owner;
+    private String petName;
+    private WeightData weightData;
+    private Weight weight;
+    private List<Weight> weightList;
+    private DateTime date;
+    private DateTime date2;
+    private String field;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'field' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5361615)